Features1/tests/stepDefinitions/newUserRegistrationContext.js

```diff
@@ -0,0 +1,73 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const { expect } = require("@playwright/test");
+
+const registerSelector = 'a[href="/register"]';
+const nameSelector = 'input[name="name"]';
+const emailSelector = 'input[name="email"]';
+const passwordSelector = 'input[name="password"]';
+const confirmPasswordSelector = 'input[name="password2"]';
+const registerButtonSelector = 'input[type="submit"]';
+const errorMsgSelector = 'div[class="alert alert-danger"]';
+const logoutSelector = 'span[class="hide-sm"]';
+
+Given("the user has navigated to the register page", async function () {
+  await page.goto("http://localhost:3000");
+  await page.click(registerSelector);
+});
+
+When(
+  "the user enters following contact information",
+  async function (dataTable) {
+    const dataArray = dataTable.hashes();
+    for (i = 0; i < dataArray.length; i++) {
+      const individualData = dataArray[i];
+      const name = individualData.name;
+      const email = individualData.email;
+      const password = individualData.password;
+      const confirmpassword = individualData.confirmpassword;
+
+      await page.type(nameSelector, name);
+      await page.type(emailSelector, email);
+      await page.type(passwordSelector, password);
+      await page.type(confirmPasswordSelector, confirmpassword);
+      await page.click(registerButtonSelector);
+    }
+  }
+);
```

Comment:
Re-use the code. This code section is same as in the Given("the user has been register with following contact information" )